src/DomBuilder/CustomerCreditTransferDomBuilder.php

```diff
@@ -93,6 +93,15 @@ public function visitPaymentInformation(PaymentInformation $paymentInformation):
         $this->currentPayment->appendChild($this->createElement('ReqdExctnDt', $paymentInformation->getDueDate()));
         $debtor = $this->createElement('Dbtr');
         $debtor->appendChild($this->createElement('Nm', $paymentInformation->getOriginName()));
+
+
+        if ($paymentInformation->getOriginAccountCurrency() === 'GBP') {
```

Comment:
So the pain format is equal for both EU & UK payments. 
For the UK market the output format differs in some places. Examples of this are;
- The need of a PstlAdr object with TwnNm & Ctry fields 
- Obviously have a different currency in the InstdAmt 
- There is a limit of 16 chars on the EndToEndId field
- The field BIC is mandatory in the CdtTrfTxInf object
- The ServiceLevel code needs to be NURG instead of SEPA
- The schemaLocation needed to be removed from the output 
- The ChrgBr field needed to be removed. 

All these requirements above made me implement them in the best way possible. I didn't want to introduce a new property in order to be fully backwards compatible.